### ISSUE TITLE
Multiplayer/Server Issues

### ISSUE THREAD

**Issue by WiredSamurai** [DuckyCrayfish/hourglass#36]
When changing the config for a server it seems to not be registering the changes properly. 1.18.2 is the version I am playing on. The config works perfectly as normal in Single-player worlds, but when transferred to the servers mod configs, it doesn't seem to make the appropriate change. The change I am referring to is to blockEntityEffect and the randomTickEffect for Crops/Livestock as well as furnaces and such. Any ideas on what I could be doing wrong, or is this a bug?

**Comment by Yukkun** [comment]
Did you change the multiplayer config? It's in different location than the single player one

**Comment by DuckyCrayfish** [comment]
Yeah @Yukkun had some good suggestions. Unfortunately I was not able to replicate the issues you are having (and not from a lack of trying). If you figure out a set of repeatable instructions I can take to reproduce the issue let me know.

Until then I am going to close this issue.
